File: 0448 find-all-numbers-disappeared-in-an-array/solution.java
class Solution {
    public List<Integer> findDisappearedNumbers(int[] nums) {
        List<Integer> l = new ArrayList<>();
        boolean arr[] = new boolean[nums.length+1];
        for (int i : nums)
            arr[i] = true;
        for (int i = 1; i < arr.length; i++)
            if (!arr[i])
                l.add(i);

        return l;
    }
}

File: 0448 find-all-numbers-disappeared-in-an-array/solution.go
func findDisappearedNumbers(nums []int) []int {
    arr:= make([]bool,len(nums))
    var a []int
    for _,v := range nums {
        arr[v-1]=true
    }
    for i,v := range arr {
        if !v {
           a = append(a,i+1)
        }
    }
    return a
}
